src/SocialGeolocationUserAdminPeopleConfigOverride.php

```diff
@@ -49,114 +49,134 @@ public function loadOverrides($names) {
     if (in_array($config_name, $names)) {
       // Grab current configuration and push the new values.
       $config = $this->configFactory->getEditable($config_name);
-      // We have to add config dependencies to field storage.
-      $dependencies = $config->getOriginal('dependencies', FALSE)['module'];
-      $dependencies[] = 'social_geolocation';
-      $dependencies[] = 'geolocation';
-      $overrides[$config_name]['dependencies']['module'] = $dependencies;
-
-      $filters = $config->getOriginal('display', FALSE)['default']['display_options']['filters'];
 
-      $filters['field_profile_geolocation_proximity'] = $this->getAdminConfigFilter();
-      $overrides[$config_name]['display']['default']['display_options']['filters'] = $filters;
+      // We have to add config dependencies to field storage.
+      $dependencies = $config->get('dependencies.module');
+      $dependencies[count($dependencies)] = 'geolocation';
```

Comment:
@OH17L Can you try using 'geolocation' as array key here and see if it works? If it does then it can solve a lot of issues for us in the distro.

```suggestion
      $dependencies['geolocation'] = 'geolocation';
```

Also don't forget to add `social_geolocation` as dependency for this configuration too.